static/app/views/settings/organizationGeneralSettings/aiFeatureSettings.tsx

```diff
@@ -1,19 +1,37 @@
 import type {FieldObject} from 'sentry/components/forms/types';
-import {t} from 'sentry/locale';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t, tct} from 'sentry/locale';
 import type {Organization} from 'sentry/types/organization';
 import {getRegionDataFromOrganization} from 'sentry/utils/regions';
 
+export const defaultEnableSeerFeaturesValue = (organization: Organization) => {
+  const isBaa = false; // TODO: add check here once we have a way to check if the org is a BAA customer. Leave it as false for now.
+  const isDeRegion = getRegionDataFromOrganization(organization)?.name === 'de';
+  return !organization.hideAiFeatures && !(isDeRegion || isBaa);
+};
+
 export const makeHideAiFeaturesField = (organization: Organization): FieldObject => {
   const isBaa = false; // TODO: add a check here once we have a way to check if the org is a BAA customer. Leave it as false for now.
   const isDeRegion = getRegionDataFromOrganization(organization)?.name === 'de';
 
   return {
     name: 'hideAiFeatures',
     type: 'boolean',
-    label: t('Hide AI Features '),
-    help: t('Hide features built with AI by default'),
-    defaultValue: isDeRegion || isBaa, // With a BAA or in the EU, we show this toggle as on when it's disabled, even though the option is false. This means we should check for the value of this field to be true.
+    label: t('Enable Seer Features'),
+    help: tct(
+      'Enables [docs:Seer features] including issue summary, autofix, and LLM-assisted search.',
+      {
+        docs: (
+          <ExternalLink href="https://docs.sentry.io/product/issues/issue-details/sentry-ai/" />
+        ),
+      }
+    ),
+    defaultValue: defaultEnableSeerFeaturesValue(organization),
     disabled: ({access}) => isDeRegion || !access.has('org:write'),
+    getValue: value => {
+      // Reversing value because the field was previously called hideAiFeatures and we've inverted the behavior.
+      return !value;
```

Comment:
lol no migration 😂 ?

Reply:
its the same thing, i think it makes sense